.git-branches.toml

```diff
@@ -0,0 +1,112 @@
+# Git Town configuration file
```

Comment:
is this default? I don't really know what all these things actually do 😓

Reply:
yea, this pretty much the default. We can always change it, they're innocuous.